network/transport/v2/senders.go

```diff
@@ -107,6 +108,20 @@ func (p *protocol) sendTransactionList(id transport.PeerID, conversationID conve
 	return nil
 }
 
+func (p *protocol) sendTransactionRangeQuery(id transport.PeerID, conversationID conversationID, lcStart uint32, lcEnd uint32) error {
```

Comment:
as with `sendTransactionListQuery`, this method is responsible for creating the conversationID.

Reply:
Seems this isn't spec'd. I'd expect it to be part of an existing conversation, because it is send after receiving `TransactionSet`? So it should use that conversationID?

6.2.4 states:
> The message MUST contain a conversationID.

But is it a new conversation, or an existing (and, which?)


https://nuts-foundation.gitbook.io/drafts/rfc/rfc017-distributed-network-grpc-v2#6.2.4-transaction-range-query

Reply:
New, only requests for TX (+payload) start a conversation a `TransactionSet` does not.

Reply:
This should probably be a new conversation. A standalone message allows faster synchronization of new nodes. Making this part of the set reconciliation loop (State / TransactionSet) would require a lot of messages to finally realize that the next step is to request the next page from a peer.

Reply:
The idea for conversations is that they span only a single req/resp. So no deep conversations. A response may contain multiple messages of the same type.